packages/wxr/src/1.3/test/objects.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+const FDBFactory = require( 'fake-indexeddb/lib/FDBFactory' );
+
+/**
+ * Internal dependencies
+ */
+const { WXRDriver } = require( '../index.js' );
+
+describe( 'Objects', () => {
+	beforeEach( () => {
+		window.indexedDB = new FDBFactory();
+	} );
+
+	test( 'Object ID increments', async () => {
+		const wxr = new WXRDriver();
+		await wxr.connect();
+
+		const first = await wxr.addObject( 'contact-form', {
+			email: 'gary@pento.net',
+			fields: [],
+		} );
+
+		const second = await wxr.addObject( 'contact-form', {
+			email: 'foo@bar.com',
+			fields: [],
+		} );
+
+		expect( first ).toEqual( 1 );
```

Comment:
From an implementation perspective it's not super useful to have the id assigned only here since the approach in  code is to [decouple the extraction](https://github.com/pento/free-as-in-speech/blob/try/add-wxr-object-support/source/services/wix/index.js#L95) from [saving to the WXR](https://github.com/pento/free-as-in-speech/blob/try/add-wxr-object-support/source/services/wix/index.js#L98).